utils/scripts/install-ubuntu.sh

```diff
@@ -14,6 +14,13 @@ set -e
 
 arch=$(uname -i)
 
+# hack to install 24.04's golang-go on 22.04:
+apt-get install -y software-properties-common
+add-apt-repository "deb http://archive.ubuntu.com/ubuntu noble main"
+apt-get update
+apt-get install -y golang-go
+add-apt-repository -r "deb http://archive.ubuntu.com/ubuntu noble main"
+
```

Comment:
Is this a part of the fix or something else?

Reply:
> Is this a part of the fix or something else?

there was a recent change that made it such that made go 1.22 a requirement and ubuntu 22 has 1.18.   Not sure why the builds didn't fail on that patch.

Reply:
This would probably be better landing as part of https://github.com/daos-stack/daos/pull/15174 where the go developers can review it.

That said, this script normally just installs packages, https://github.com/daos-stack/daos/blob/master/utils/docker/Dockerfile.ubuntu would be a better place for this code, or perhaps a `utils/scripts/helpers/repo-helper-debian.sh` script to match what rocky does.